Dockerfile

```diff
@@ -22,6 +22,12 @@ RUN cd /usr/src/nginx-${NGINX_VERSION}/ && ./configure --add-module=/usr/src/ngx
 RUN cd /usr/src/nginx-${NGINX_VERSION}/ && make
 RUN cd /usr/src/nginx-${NGINX_VERSION}/ && sudo make install
 
+#cleanup
+RUN rm -fr /usr/src/*
+RUN apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+    find /var/log -type f | while read f; do echo -ne '' > $f; done;
+
```

Comment:
@unlucio would be good if the line cleanup commands are the last RUN command before CMD